README.md

```diff
@@ -1,6 +1,7 @@
 # Homely - Storage Queues library.
 
-[![Build Status](https://dev.azure.com/homelyau/Homely.Storage.Queues/_apis/build/status/Homely.Homely.Storage.Queues?branchName=master)](https://dev.azure.com/homelyau/Homely.Storage.Queues/_build/latest?definitionId=7&branchName=master)
+[![Build Status](https://dev.azure.com/homelyau/Homely.Storage.Queues/_apis/build/status/Homely.Homely.Storage.Queues?branchName=refs%2Fpull%2F5%2Fmerge)](https://dev.azure.com/homelyau/Homely.Storage.Queues/_build/latest?definitionId=7&branchName=refs%2Fpull%2F5%2Fmerge)

```

Comment:
just checking .... branchName == refs/pull/merge?  vs  master?

Reply:
Came from the pipelines UI (ie, i didn't do manually) ..so assume it's correct?

![image](https://user-images.githubusercontent.com/930343/110038711-399aa500-7d94-11eb-810f-7878228cd621.png)